Mage/src/main/java/mage/abilities/dynamicvalue/common/ControlledCreaturesGreatestPower.java

```diff
@@ -0,0 +1,41 @@
+package mage.abilities.dynamicvalue.common;
+
+import mage.abilities.Ability;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+/**
+ * @author jimga150
+ */
+public enum ControlledCreaturesGreatestPower implements DynamicValue {
```

Comment:
GreatestPowerAmongControlledCreaturesValue already exists

(and is a good example of stream, mapToInt, sum as a cleaner style)